src/EnergyPlus/Furnaces.hh

```diff
@@ -142,48 +142,41 @@ namespace Furnaces {
-        Real64 DesignSuppHeatingCapacity;               // Nominal Capacity of Supplemental Heating Coil [W]
-        Real64 DesignFanVolFlowRate;                    // Vol Flow through the Furnace being Simulated [m**3/Sec]
-        bool DesignFanVolFlowRateEMSOverrideOn;         // if true, then EMS is calling to override autosize fan flow
-        Real64 DesignFanVolFlowRateEMSOverrideValue;    // EMS value for override of fan flow rate autosize [m3/s]
-        Real64 DesignMassFlowRate;                      // Design mass flow rate through furnace [kg/s]
-        Real64 MaxCoolAirVolFlow;                       // supply air volumetric flow rate during cooling operation [m3/s]
-        bool MaxCoolAirVolFlowEMSOverrideOn;            // if true, EMS is calling to override autosize flow during cooling
-        Real64 MaxCoolAirVolFlowEMSOverrideValue;       // EMS value for override of flow during cooling [m3/s]
-        Real64 MaxHeatAirVolFlow;                       // supply air volumetric flow rate during cooling operation [m3/s]
-        bool MaxHeatAirVolFlowEMSOverrideOn;            // if true, EMS is calling to override autosize flow during heating
-        Real64 MaxHeatAirVolFlowEMSOverrideValue;       // EMS value for override of flow during heating operation [m3/s]
-        Real64 MaxNoCoolHeatAirVolFlow;                 // supply air volumetric flow rate when no cooling or heating [m3/s]
-        bool MaxNoCoolHeatAirVolFlowEMSOverrideOn;      // if true, EMS is calling to override autosize no heatcool rate
-        Real64 MaxNoCoolHeatAirVolFlowEMSOverrideValue; // EMS value for override of flow during no heat cool [m3/s]
-        Real64 MaxCoolAirMassFlow;                      // supply air mass flow rate during cooling operation [kg/s]
-        Real64 MaxHeatAirMassFlow;                      // supply air mass flow rate during heating operation [kg/s]
-        Real64 MaxNoCoolHeatAirMassFlow;                // supply air mass flow rate when no cooling or heating [kg/s]
-        Real64 MaxHeatCoilFluidFlow;                    // water or steam mass flow rate for heating coil [kg/s]
-        Real64 MaxSuppCoilFluidFlow;                    // water or steam mass flow rate for supplemental heating coil [kg/s]
-        Real64 ControlZoneMassFlowFrac;                 // Fraction of furnace flow to control zone
-        Real64 DesignMaxOutletTemp;                     // Maximum supply air temperature from furnace heater [C]
-        Real64 MdotFurnace;                             // Mass flow rate through furnace [kg/s]
-        Real64 FanPartLoadRatio;                        // Part load ratio of furnace fan (mdot actual/mdot design)
-        Real64 CompPartLoadRatio;                       // Part load ratio of furnace compressor (load / steady-state output)
-        Real64 WSHPRuntimeFrac;                         // Runtime fraction of water source heat pump
```

Comment:
Code changes seem to be working well based heavily on CI results and local testing.